### ISSUE TITLE
Needs explanation about the  train and test results

### ISSUE THREAD

**Issue by anwaar0** [criteo-research/pytorch-ada#19]
Currently, I am using this library, I found these test results during testing for MNIST-USPS digit domain adaptation. 

TEST RESULTS
{'Te_domain_acc': 0.29888888888888887,
'Te_source_acc': 0.9421212121212121,
'Te_target_acc': 0.1322222222222222,
'avg_test_loss': 1.5861893892288208,
'test_loss': 1.5861893892288208}

Can you explain the above short form of evaluation metrics? What I understood is
Te_domain_acc mean test domain accuracy 
Te_source_acc mean test source accuracy 
Te_target_acc mean test target accuracy 
avg_test_loss mean average test loss
test_loss mean test loss

can you briefly describe the above, especially about Te_domain_acc in terms of MNIST-USPS?

Further, I also need a description of the table given below. I got this table after the completion of training.
method	source acc		target acc		domain acc
CDAN      98.5% +- 0.25		90.1% +- 1.70		42.3% +- 2.63 (Validation)
CDAN      98.5% +- 0.24		90.6% +- 1.78		41.8% +- 2.56 (Test)
CDAN-E    98.6% +- 0.14		92.0% +- 0.79		33.8% +- 6.07 (Validation)
CDAN-E    98.6% +- 0.18		92.5% +- 1.34		34.0% +- 5.52 (Test)
DAN       98.8% +- 0.14		95.1% +- 0.58		nan% +- nan (Validation)
DAN       98.9% +- 0.13		95.5% +- 0.40		nan% +- nan (Test)
DANN      98.9% +- 0.19		93.1% +- 1.49		46.9% +- 18.23 (Validation)
DANN      98.9% +- 0.18		93.6% +- 0.89		46.3% +- 18.16 (Test)
JAN       98.6% +- 0.10		91.7% +- 0.97		nan% +- nan (Validation)
JAN       98.6% +- 0.17		91.7% +- 0.87		nan% +- nan (Test)
Source    98.9% +- 0.12		93.5% +- 1.41		72.4% +- 11.27 (Validation)
Source    98.9% +- 0.17		94.0% +- 0.75		71.6% +- 11.04 (Test)
WDGRL     97.2% +- 0.73		74.7% +- 7.56		54.9% +- 22.12 (Validation)
WDGRL     97.1% +- 0.74		73.7% +- 7.91		54.7% +- 21.84 (Test)

I understand that different methods are applied to source and target datasets. However, I still have a few questions,
Is this library using the pre-trained weights of MINIST   (source)?
What is 42.3% +_2.63 in the domain ACC (first row and first column )?

I know, my issue may be general but it will be great if someone can provide information about these results.
Thank you.
